addons/html_editor/static/src/fields/html_field.xml

```diff
@@ -1,12 +1,15 @@
 <templates xml:space="preserve">
     <t t-name="html_editor.HtmlField">
-        <t t-if="props.readonly">
-            <div  t-ref="readonlyElement" class="o_readonly" t-out="props.record.data[this.props.name]" />
+        <t t-if="this.displayReadonly">
+            <HtmlViewer
+                value="props.record.data[props.name]"
+                cssAssetId="props.cssReadonlyAssetId"
```

Comment:
probably need a `t-key="'' + props.cssReadonlyAssetId + sandboxedPreview"`. If these props change, the htmlviewer will behave badly (not doing what we might expect => reloading assets and stuff)